schedule/yast/addon-module-http.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: addon-module-http
+description: |
+  Test verifies that adding addon using http works fine. We just perform installation
+  here. Using HA here, related ticket: https://progress.opensuse.org/issues/25896
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - '{{hostname_inst}}'
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - '{{reconnect_mgmt_console}}'
+  - '{{grub_test}}'
+  - installation/first_boot
+conditional_schedule:
+  hostname_inst:
+    BACKEND:
+      qemu:
+        - installation/hostname_inst
+      svirt:
+        - installation/hostname_inst
+  reconnect_mgmt_console:
+    MACHINE:
+      ppc64le-hmc-single-disk:
```

Comment:
Using `MACHINE` variable doesn't scale well as might get renamed. We can put both modules together in the `BACKEND` condition. And for `svirt` backend we can define just `boot/reconnect_mgmt_console` to handle s390x case.

Reply:
Done